src/auth/strategies/oidc.strategy.ts

```diff
@@ -43,17 +44,17 @@ export class OidcStrategy extends PassportStrategy(Strategy, "oidc") {
     private configService: ConfigService,
     private usersService: UsersService,
   ) {
-    const oidcConfig = configService.get<Record<string, unknown>>("oidc");
+    const oidcConfig = configService.get<OidcConfig>("oidc");
     super({
       client: client,
       params: {
         redirect_uri: oidcConfig?.callbackURL,
         scope: oidcConfig?.scope,
-        authorization_endpoint: oidcConfig?.authorizationURL,
```

Comment:
How does the OidcStrategy know what OIDC server to talk to wihtout the authorizationURL?

Reply:
from memory, I think the `?` is because oidcConfig which is the result of getConfig can be undefined. I'll try to check better in the next days

Reply:
In src/auth/strategies/oidc.strategy.ts Issuer is built by providing a configuration from a remote server. 
```
const trustIssuer = await Issuer.discover(
      `${oidcConfig?.issuer}/.well-known/openid-configuration`,
    );
```
This configuration has the `authorizationURL`.